examples/hello-world/package.json

```diff
@@ -15,8 +15,20 @@
     "build:dist10": "lb-tsc es2018",
     "clean": "lb-clean *example-hello-world*.tgz dist* package api-docs",
     "verify": "npm pack && tar xf *example-hello-world*.tgz && tree package && npm run clean",
+    "lint": "npm run prettier:check && npm run tslint",
+    "lint:fix": "npm run prettier:fix && npm run tslint:fix",
+    "prettier:cli": "lb-prettier \"**/*.ts\" \"**/*.js\"",
+    "prettier:check": "npm run prettier:cli -- -l",
+    "prettier:fix": "npm run prettier:cli -- --write",
+    "tslint": "lb-tslint",
+    "tslint:fix": "npm run tslint -- --fix",
+    "pretest": "npm run clean && npm run build:current",
+    "test": "lb-mocha --allow-console-logs \"DIST/test\"",
+    "posttest": "npm run lint",
+    "test:dev": "lb-dist mocha DIST/test/**/*.js && npm run lint",
```

Comment:
I think we should be using `lb-mocha` instead of `lb-dist mocha` in all `test:dev` scripts, thoughts?

Reply:
agreed